### DISCUSSION TITLE
Report file name in case of parsing error

### DISCUSSION THREAD

**Discussion by gunnarmorling** [duckdb/duckdb#5959] [Ideas]
I'm using `read_csv()` to query a set of CSV files in a directory:

```
SELECT COUNT(*) FROM read_csv('my-csvs/*.csv', ...);
```

One of the files has a datum which doesn't correspond to the configured format, so the query fails:

```
Error: Invalid Input Error: Could not parse string "18-1218-12-1991" according to format specifier "%d-%m-%Y"
```

Suggestion: Please print out the name of the affected file. While I could grep for the problematic file, having the the name as part of the error message will streamline the experience. Thanks a lot for your consideration!

**Comment by Mytherin** [comment]
Thanks for the report - I agree this should be fixed. I have created an issue out of this discussion (#6084).

**Reply by Mytherin** [reply]
Looks like this one has already been fixed in the latest master
